code/modules/materials/material_sheets.dm

```diff
@@ -314,7 +314,7 @@
 	if(plank_type && (IS_HATCHET(W) || IS_SAW(W)))
 		var/tool_type = W.get_tool_quality(TOOL_HATCHET) >= W.get_tool_quality(TOOL_SAW) ? TOOL_HATCHET : TOOL_SAW
 		if(W.do_tool_interaction(tool_type, user, src, 1 SECOND, set_cooldown = TRUE) && !QDELETED(src))
-			var/obj/item/stack/planks = new plank_type(loc, 1, material?.type, reinf_material?.type)
+			var/obj/item/stack/planks = new plank_type(loc, rand(2,4), material?.type, reinf_material?.type) // todo: change plank amount based on carpentry skillcheck
 			planks.add_to_stacks(user, TRUE)
```

Comment:
```suggestion
			var/obj/item/stack/planks = new plank_type(get_turf(src), rand(2,4), material?.type, reinf_material?.type) // todo: change plank amount based on carpentry skillcheck
			if(!planks.add_to_stacks(user, TRUE))
				user.put_in_hands(planks)
```
The put_in_hands part is optional, but as-is, using `loc` means planks cut from held logs will appear inside your body.